### DISCUSSION TITLE
French translation : improve text

### DISCUSSION THREAD

**Discussion by Pieloth** [seerge/g-helper#1751] [Ideas]
Hi,

Thanks for this very good app for Asus ROG laptop (G17-2023 G713PV)

I would suggest a tiny modification for the French language translation, which is not exactly translated IMHO, and makes it a bit confusing to understand:

In [g-helper](https://github.com/seerge/g-helper/tree/main)/[app](https://github.com/seerge/g-helper/tree/main/app)/[Properties](https://github.com/seerge/g-helper/tree/main/app/Properties)/Strings.fr.resx

Currently text is for keyboard backlight timeout: 
```
  <data name="BacklightTimeout" xml:space="preserve">
    <value>Délai d'activation de la batterie (0 - ON)</value>
  </data>
```

Improved French translation proposed, to better match original English text without ambiguity:
```
  <data name="BacklightTimeout" xml:space="preserve">
    <value>Délai branché / sur batterie (0 - ON)</value>
  </data>
```

**Comment by seerge** [comment]
@Pieloth hello, all translations are "managed" via Crowdin https://github.com/seerge/g-helper/discussions/759

You can just join project there, and make your changes :) they will get merged automatically in next releases

**Comment by Pieloth** [comment]
Ok, thanks @seerge, my mistake, did not see this post while searching for translation in discussions

I will join there the Crowdin project
